app/commands/user/insiders_status/trigger_update.rb

```diff
@@ -11,6 +11,6 @@ def call
       user.update!(insiders_status: :unset) if user.insiders_status_ineligible?
     end
 
-    User::InsidersStatus::Update.defer(user)
+    User::InsidersStatus::Update.(user)
```

Comment:
I think this one should generally be `defer` actually. But we might just an exception for the donation route. If you revert this for now, I'll work it out when I test everything e2e properly.